sample_files/problem_setting/examples/identical_checker.cpp

```diff
@@ -0,0 +1,132 @@
+#include <algorithm>
+#include <cstdio>
+#include <cstdlib>
+#include <fstream>
+#include <numeric>
+#include <regex.h>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+namespace CheckerCodes {
+int AC = 0;
+int WA = 1;
+int PE = 2;
+int PARTIAL = 7;
+} // namespace CheckerCodes
```

Comment:
Sounds like this should be an `enum`.

Reply:
If we use a bare enum, then `AC, WA, PE, PARTIAL` leak into the symbol table, and if we use an `enum class`, then `std::exit(CheckerCodes::AC)` doesn't work

Reply:
If you don't want the enum to leak into the global scope, then declare it in a `namespace`. Don't declare a bunch of random mutable global variables. You can actually do `CheckerCodes::AC = 1337;` like this.

Reply:
I feel like it's best to just use constexpr and be done with it, no?

I dont see what you mean with `CheckerCodes::AC = 1337`, TMK that requires a previous declaration of the variable(?)

Reply:
Changed to constexpr